src/daos/sql-query-builder.php

```diff
@@ -112,15 +126,17 @@ public function or_where( $string ) {
 
   /**
    * Adds a variable to be bound to in the prepared statement
+   *
    * @param $name : string
    * @param $value : object
    */
-  public function bind_value( $name, $value ) {
-    $this->values_to_bind[ $name ] = $value;
+  public function bind_value( $name, $value, $data_type = false ) {
+    $this->values_to_bind[ $name ] = array( $value, $data_type );
```

Comment:
Lets use named array indexes so its clear whats what. I know this is a simple case but its worth it for readability to add a few characters: 
`[ 'value' => $value, 'data_type' => $data_type]`